week_5/week5/src/main/java/org/example/InputMismatchException.java

```diff
@@ -0,0 +1,24 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class InputMismatchException {
+    public static void main(String[] args) {
+        System.out.println("Enter two numbers:");
+        validInput();
+    }
+    static void validInput(){
+        Scanner scanner=new Scanner(System.in);
+        try {
+
+            int a= scanner.nextInt();
+            int b= scanner.nextInt();
+            System.out.println("sum="+(a+b));
+
+        }catch (java.util.InputMismatchException e){
+            System.out.println("you entered wrong input please enter input again:");
+            validInput();
```

Comment:
what about  comma or if there is "salah-mohamed" for this text your code is correct but think about the cases i mentioned